### ISSUE TITLE
Hanzi Graph Proportionate Representation of Increase

### ISSUE THREAD

**Issue by Nick3C** [batterseapower/pinyin-toolkit#73]
At some point in the future... not before 0.05 as it is unnecessary :)

We should consider a way to show the increase of the zoomed graphs by the type of character added. For example on zoomed graph with many characters you will almost certainly only see the top (advanced hanzi).

It would be nice to see the increase in types on hanzi in this graph. For example:
start period = 800 character
end period = 900 characters
increase in basic = 50 character
increase in ele = 10 char
increase in int =5 char
increase in adv = 2
increase in non = 1

If you look at the graph it will all be blue because non-hsk characters are at the top and any increase is "hidden" by the zoom.

This will likely be tough to do. I guess the bet thing would be to draw the colours as a % of whatever is being displayed.

**Comment by batterseapower** [comment]
Yes, this does seem like it would be helpful. One option would be to have one graph per HSK level, but it would possibly add a lot of clutter.

**Comment by Nick3C** [comment]
I have been thinking about this. Actually we don't really want a proportional increase, what we want is to get the number of each level at the start of the range, eg:
day -7 = 500, 500, 300, 100, 50

and then generate the graphs based on numbers above this. eg:
day -6 added 3,2,3,2,0
then add graphs according to this rather than just seeing the top part of the graph.

I think that makes more sense because it shows what is actually being added rather than just showing us the top of the graph. If we look at a long-range then it will show the same graph as it currently does (becase cards are 0,0,0,0,0

**Comment by Nick3C** [comment]
User commentary on this:
https://groups.google.com/group/ankisrs/browse_thread/thread/154f2a84361d78b8#
